test/ufuzz.js

```diff
@@ -11,6 +11,79 @@
 var vm = require("vm");
 var minify = require("..").minify;
 
+var MAX_GENERATED_FUNCTIONS_PER_RUN = 1;
+var MAX_GENERATION_RECURSION_DEPTH = 15;
+var INTERVAL_COUNT = 100;
+
+var VALUES = [
+  'true',
+  'false',
+  '22',
+  '0',
+  '-0', // 0/-0 !== 0
+  '23..toString()',
+  '24 .toString()',
+  '25. ',
+  '0x26.toString()',
+  '(-1)',
+  'NaN',
+  'undefined',
+  'Infinity',
+  'null',
+  '[]',
+  '({})', // unwrapped the object causes too many syntax errors in statements
+  '"foo"',
+  '"bar"' ];
+
+var BINARY_OPS_NO_COMMA = [
+  ' + ',
+  ' - ',
+  '/',
+  '*',
+  '&&',
+  '||',
+  '^' ];
+
+var BINARY_OPS = [','].concat(BINARY_OPS_NO_COMMA);
+
+var ASSIGNMENTS = [
+  '=',
+  '=',
+  '=',
+  '-=',
+  '/=',
+  '*=',
+  '&=',
+  '|=',
+  '^=',
+  '+=' ];
```

Comment:
I think this is more a parser thing. We could add all the compound ops but they are very probably treated equally as a group. Unless you say that's not the case. In that case every compound op should be added there.